apps/web/lib/settings/left-side-setting-menu.tsx

```diff
@@ -99,14 +99,9 @@ export const LeftSideSettingMenu = ({ className }: { className?: string }) => {
 								: 'border-l-transparent font-normal dark:text-[#7E7991]'
 						}
 						`}
-						wrapperClassName={`w-full border-t-0 border-r-0 border-b-0 rounded-none
-                font-normal text-[#7e7991] justify-start  pt-[24px] pb-[24px] pl-[24px]
-				border-l-[5px] ${
-					activePage === '/settings/personal'
```

Comment:
@Cedric921 can you check this PR, why we have this conditional styling and now it's removed?

Reply:
@Cedric921 and @evereq , I'd noticed that the style shouldn't be conditioned by the "activePage" variable but with the "open" variable that headlessUI makes available to us in the SidebarAccordian component, so I just moved the code around and didn't delete anything.